### DISCUSSION TITLE
Splitscreen

### DISCUSSION THREAD

**Discussion by zacnmac** [Techokami/SonicWorldsNext#84] [Ideas]
split screen would be cool so you can split up while playing two players, as opposed to tails just always being left behind.

**Comment by DimensionWarped** [comment]
It would be cool, but it'd also be a lot of work and it's not really in scope as-is... hell, it's kinda hard to justify all the stuff we've done around partner characters in general when you consider how few people actually play fangames with a friend.

If you can come up with a method and make it a disabled by default toggle we'd probably be willing to take it though.

**Reply by Raptorq1** [reply]
I have already implemented split screen play for my game and it works just fine
![Screenshot_20230915-184039](https://github.com/Techokami/SonicWorldsNext/assets/106418027/59a2466a-a204-4166-a322-286f43db18d9)

**Comment by DimensionWarped** [comment]
Alright, well feel free to make a pull request as long as it doesn't break anything and it's toggleable. I was thinking something a little more elaborate like the screen splitting at a certain distance myself, but this the only thing particularly missing here is vertical scaling of the viewport.

**Reply by Raptorq1** [reply]
I was the one that made the split screen 2player screen

Heres the tutorial I used for this


https://youtu.be/tkBgYD0R8R4?si=PXk2DymONUn8ct0y
